Update docs to use ARM based images for Apple M1/M2 Macbooks (#45142)
ray-project/ray: doc/source/cluster/kubernetes/user-guides/config.md

> If you are using a nightly or development Ray image, it is fine to specify Ray's

Reviewer: ```suggestion
If you are using a nightly or development Ray image, you can specify Ray's
```

latest release version under `spec.rayVersion`.

> For Apple M1/M2 Macbooks, please refer to [this troubleshooting doc](docker-image-for-apple-macbooks) on specifying the 

Reviewer: ```suggestion
For Apple M1 or M2 MacBooks, see [Use ARM-based docker images for Apple M1 or M2 MacBooks](docker-image-for-apple-macbooks) to specify the 
```

> right image.

Reviewer: ```suggestion
correct image.
```

> Code dependencies for a given Ray task or actor must be installed on each Ray node that

Reviewer: ```suggestion
You must install code dependencies for a given Ray task or actor on each Ray node that
```

might run the task or actor.

> To achieve this, it is simplest to use the same Ray image for the Ray head and all worker groups.

Reviewer: ```suggestion
The simplest way to achieve this configuration is to use the same Ray image for the Ray head and all worker groups.
```
